test/specs/core.element.tests.js

```diff
@@ -42,10 +42,10 @@ describe('Core element tests', function() {
 			colorProp: 'rgb(64, 64, 0)',
 		});
 
-		// Final transition clones model into view
+		// Final transition does not close _model to _view anymore
```

Comment:
Why does this behavior change? Does it have any user visible effect?

Also, I'm guessing you mean "clone" instead of "close". I'm not sure it's useful to have a comment describing what the behavior used to be though, so I might remove just remove the comment

Reply:
Its because I removed the `if (!model || ease === 1) {` clone.
It came from #6129 and that case seems to work just fine with this PR.

What I meant with "break the animations" - I did not check earlier if the issue fixed by those removed lines has resurfaced. Now I have and it didn't.